api_tests/files/views/test_file_detail.py

```diff
@@ -23,8 +23,7 @@
 def _dt_to_iso8601(value):
     iso8601 = value.isoformat()
     if iso8601.endswith('+00:00'):
-        iso8601 = iso8601[:-9] + 'Z'  # offset upped to 9 to get rid of 3 ms decimal points
-
+        iso8601 = iso8601[:-6] + 'Z'  # offset upped to 9 to get rid of 3 ms decimal points
```

Comment:
I've updated the comment and kept the decimal places. This was a change introduce in the new version of DRF. The rationale is that JSON is not a strict subset of ECMAScript which stipulates that dates look this way.